### PR TITLE
Add Vehkamaki sulfate nucleation tests

### PR DESCRIPTION
**Pull request by fvitt**
Add tests for Vehkamaki sulfate nucleation

### CODE REVIEW

**Review by drmikemills — APPROVED** [review]

File: tests/Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ TEST_OBJ = carma_bcoctest.o carma_bc2gtest.o carma_coagtest.o carma_drydeptest.o
 carma_growtest.o carma_inittest.o carma_kappawetrtest.o carma_mietest.o carma_nuctest.o \
 carma_pheattest.o carma_sigmadrydeptest.o carma_sigmafalltest.o carma_swelltest.o \
 carma_vdiftest.o carma_test.o atmosphere_mod.o carma_scfalltest.o carma_growsubtest.o \
-carma_growintest.o carma_fractalmicrotest.o carma_fractalopticstest.o carma_sulfhettests.o
+carma_growintest.o carma_fractalmicrotest.o carma_fractalopticstest.o carma_sulfhettest.o
```
Comment (cbardeen) [inline tests/Makefile:14]: Don't you also need to add the 2 new tests here?

File: tests/carma_sulfate_vehkamaki_test.F90 [hunk 2]
```diff
@@ -0,0 +1,380 @@
+!! This code is to test the impact of particle swelling from
```
Comment (cbardeen) [inline tests/carma_sulfate_vehkamaki_test.F90:1]: This description should be updated with the actual purpose of this test. This is a copy and past error that may be in several of the tests.

**Review by cbardeen — CHANGES_REQUESTED** [review]

File: tests/Makefile (hunk 1 above)
Comment (fvitt) [reply tests/Makefile]: fixed

File: tests/carma_sulfate_vehkamaki_test.F90 (hunk 2 above)
Comment (fvitt) [reply tests/carma_sulfate_vehkamaki_test.F90]: fixed

**Pull request merged**